src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+		case 'X': return __tr2qs("X: Special powers"); break;
+		case 'h': return __tr2qs("h: Host changed"); break;
+	}
+	return KviIrcuIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviDarenetIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'F': return __tr2qs("F: Disable fake lag"); break;
+		case 'H': return __tr2qs("H: Hide oper status"); break;
+		case 'N': return __tr2qs("N: Network administrator"); break;
+		case 'P': return __tr2qs("P: Recipient for notices on WHOIS"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'X': return __tr2qs("X: Special powers"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
+	}
+	return KviIrcuIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviHybridServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'D': return __tr2qs("D: Deaf"); break;
+		case 'F': return __tr2qs("F: Recipient for far connect/quit notices"); break;
+		case 'G': return __tr2qs("G: Only accept private messages from users who share common channels"); break;
+		case 'H': return __tr2qs("H: Hide oper status"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Connected over SSL"); break;
+		case 'W': return __tr2qs("W: Connected over WEBIRC"); break;
+		case 'a': return __tr2qs("a: Server administrator"); break;
+		case 'b': return __tr2qs("b: Recipient for bot/join flood warnings"); break;
+		case 'e': return __tr2qs("e: Recipient for server introduction and split notices"); break;
+		case 'f': return __tr2qs("f: Recipient for full I-Line notices"); break;
+		case 'g': return __tr2qs("g: Only allow accepted clients to message you"); break;
+		case 'j': return __tr2qs("j: Recipient for rejected client notices"); break;
+		case 'l': return __tr2qs("l: Recipient for LOCOPS"); break;
+		case 'p': return __tr2qs("p: Channels hidden from WHOIS"); break;
+		case 'q': return __tr2qs("q: Idle time hidden from WHOIS"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 'u': return __tr2qs("u: Recipient for unauthorized client notices"); break;
+		case 'x': return __tr2qs("x: Host hidden"); break;
+		case 'y': return __tr2qs("y: Can see stats/links/admin requests"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviIrcdSevenIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'O': return __tr2qs("O: IRC help operator"); break;
+		case 'Q': return __tr2qs("Q: Prevents you from being affected by channel forwarding"); break;
+		case 'S': return __tr2qs("S: Network service"); break;
+		case 'Z': return __tr2qs("Z: Connected over SSL"); break;
+		case 'h': return __tr2qs("h: Marks you as a helper in /stats p"); break;
+		case 'p': return __tr2qs("p: Enable oper overrides"); break;
```

Comment:
Above you chose to capitalize OPER, this should be carried over consistently. either you capitalize all instances of the same or you dont.

My preference is the capitalized version, but again I leave that to you, as long as its consistent.